test/micro/org/openjdk/bench/java/lang/MathBench.java

```diff
@@ -62,13 +64,46 @@ public class MathBench {
     public float float1 = 1.0f, float2 = 2.0f, floatNegative99 = -99.0f, float7 = 7.0f, eFloat = 2.718f;
     public double double1 = 1.0d, double2 = 2.0d, double81 = 81.0d, doubleNegative12 = -12.0d, double4Dot1 = 4.1d, double0Dot5 = 0.5d;
 
+    @Param({"-2.0", "-1.0", "-0.5", "-0.1", "0.0", "0.1", "0.5", "1.0", "2.0"})
+    public double tanhConstInput;
```

Comment:
Did you check if these values constant fold ?
The field is not a static final.

Reply:
Thanks for pointing this out. They should now be constant values. If there are constants required, please let me know.